worker/src/test/scala/org/apache/celeborn/service/deploy/MiniClusterFeature.scala

```diff
@@ -119,19 +119,19 @@ trait MiniClusterFeature extends Logging {
     // shutdown workers
     workerInfos.foreach {
       case (worker, _) =>
-        worker.close()
+        worker.stop(false)
         worker.rpcEnv.shutdown()
     }
 
     // shutdown masters
-    masterInfo._1.close()
+    masterInfo._1.stop(false)
     masterInfo._1.rpcEnv.shutdown()
 
     // interrupt threads
     Thread.sleep(5000)
     workerInfos.foreach {
       case (worker, thread) =>
-        worker.shutdown(graceful = false)
+        worker.stop(false)
```

Comment:
we don't need to stop the worker once again, it was introduced in https://github.com/apache/incubator-celeborn/pull/1727, let's revert it